lib/Ride.dart

```diff
@@ -182,37 +164,38 @@ class _RideCardState extends State<RideCard> {
                       Container(
                         decoration: BoxDecoration(
                           shape: BoxShape.circle,
-                          boxShadow: [dropShadow],
+                          boxShadow: [CarriageTheme.shadow],
                         ),
                         child: CircleAvatar(
                           radius: 24,
                           //TODO: replace with rider's image
                           backgroundImage:
-                              AssetImage('assets/images/terry.jpg'),
+                          AssetImage('assets/images/terry.jpg'),
                         ),
                       ),
                       SizedBox(width: 16),
                       Column(
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: [
                             Text(widget.ride.rider.firstName,
-                                style: TextStyle(
-                                    fontSize: 20, fontWeight: FontWeight.bold)),
+                                style: CarriageTheme.title3
+                            ),
                             SizedBox(height: 4),
-                            widget.ride.rider.accessibilityNeeds.length > 0
-                                ? Text(
-                                    widget.ride.rider.accessibilityNeeds
-                                        .join(', '),
-                                    style: TextStyle(
-                                        color: Color(0xFF848484),
-                                        fontStyle: FontStyle.italic,
-                                        fontSize: 15))
-                                : Container()
-                          ]),
+                            widget.ride.rider.accessibilityNeeds.length > 0 ?
+                            Text(
+                                widget.ride.rider.accessibilityNeeds.join(', '),
+                                style: TextStyle(
+                                    color: Color(0xFF848484),
```

Comment:
Can we put this color in the theme? I think it's been used > 1 time

Reply:
Will add when profile page is updated to use the design system